environment.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
   - zipp=3.4.0
   - zope.interface=5.2.0
   - conda-forge::astropy>=1.3
-  - conda-forge::numpy>=1.15.0  # See CATKIT-91
+  - conda-forge::numpy>=1.19.2  # See CATKIT-91
```

Comment:
Whoops, I should have paid better attention from the beginning. I totally missed the ``>``. I thought I had double-checked that this used 1.19.2 but doing so again it didn't it used 0.20.1.

Reply:
Actually, scratch that, ``origin/develop`` was using 1.19.2 as we can see [here](https://github.com/spacetelescope/catkit/runs/2176266394#step:5:503) but ``photutils=1.1.0`` ([here](https://github.com/spacetelescope/catkit/runs/2176266394#step:5:310)).

Using ``numpy=1.17.5`` also failed and picked up ``photutils=1.1.0``.